searx/webapp.py

```diff
@@ -164,6 +164,14 @@ def get_locale():
     return locale
 
 
+def get_doi_resolver():
```

Comment:
Seems this function is identical to `searx/plugins/oa_doi_rewrite.py:get_doi_resolver()`. You could import that function to avoid code duplication.

Reply:
Sure!